src/data/nat/factorization/basic.lean

```diff
@@ -481,6 +481,13 @@ begin
   { simp [hqp, (factorization_le_iff_dvd hd0 hn0).2 hdn q] },
 end
 
+/-- If `n` is a nonzero natural number and `p` is a prime, then there are natural numbers `e`
+and `n'` such that `n'` is not divisible by `p` and `n = p^e * n'`. -/
+lemma prime_power_and_complement {n : ℕ} (hn : n ≠ 0) (p : ℕ) [fact p.prime] :
```

Comment:
Maybe
```suggestion
lemma exists_prime_power_and_complement {n : ℕ} (hn : n ≠ 0) (p : ℕ) [fact p.prime] :
```
or
```suggestion
lemma eq_prime_power_mul_complement {n : ℕ} (hn : n ≠ 0) (p : ℕ) [fact p.prime] :
```

Reply:
Maybe `exists_prime_pow_and_compl`? (Seeing that `pow` is the usual spelling for powers in names and the complement is referred to as `compl` elsewhere in the file. I'll change it to this, but feel free to comment if you don't like it.

Reply:
Your suggestion looks good to me.